its/ruling/src/test/java/org/sonarsource/iac/IacRulingTest.java

```diff
@@ -44,7 +44,7 @@
 class IacRulingTest {
 
   private static final String SQ_VERSION_PROPERTY = "sonar.runtimeVersion";
-  private static final String DEFAULT_SQ_VERSION = "LATEST_RELEASE[8.9]";
+  private static final String DEFAULT_SQ_VERSION = "LATEST_RELEASE";
```

Comment:
I remember you explaining to me this, but I forgot.
So, what does the `[8.9]` in `LATEST_RELEASE[8.9]` mean?

Reply:
It will pick the latest release of the 8.9 branch, which is the LTS branch. So 8.9.1 -> 8.9.2 -> 8.9.3 and so on.
However, during the CI pipe this version will be replaced by the CI task. This field is only taken on local runs.